src/vs/editor/contrib/suggest/browser/suggestWidgetRenderer.ts

```diff
@@ -213,7 +213,8 @@ export class ItemRenderer implements IListRenderer<CompletionItem, ISuggestionTe
 			data.detailsLabel.textContent = stripNewLines(completion.detail || '');
 			data.root.classList.add('string-label');
 		} else {
-			data.parametersLabel.textContent = stripNewLines(completion.label.detail || '');
+			const labelDetail = stripNewLines(completion.label.detail || '');
+			data.parametersLabel.textContent = normalizeLabelDetail(labelDetail);
```

Comment:
We're sure this bug applies to the editor too? That would be surprising. Better to get a review from Joh if so. Also this doesn't seem important enough for October, especially if it's an issue in the editor.